monai/engines/trainer.py

```
@@ -76,10 +83,144 @@ def get_stats(self, *vars):
+        compile_kwargs: keyword arguments forwarded to ``torch.compile()``.
+        **kwargs: remaining arguments forwarded to ``Trainer`` / ``Workflow``.
+    """
+
+    def __init__(
+        self,
+        network: torch.nn.Module,
+        optimizer: Optimizer,
+        inferer: Inferer | None = None,
+        optim_set_to_none: bool = False,
+        accumulation_steps: int = 1,
+        compile: bool = False,
```

Ruff (0.15.18)

[error] Function argument `compile` is shadowing a Python builtin

(A002)